caravel/models.py

```diff
@@ -2005,10 +2011,73 @@ def to_dict(self):
             'tempTable': self.tmp_table_name,
             'userId': self.user_id,
         }
+
     @property
     def name(self):
         ts = datetime.now().isoformat()
         ts = ts.replace('-', '').replace(':', '').split('.')[0]
         tab = self.tab_name.replace(' ', '_').lower() if self.tab_name else 'notab'
         tab = re.sub(r'\W+', '', tab)
         return "sqllab_{tab}_{ts}".format(**locals())
+
+
+class DatasourceAccessRequest(Model, AuditMixinNullable):
+    """ORM model for the access requests for datasources and dbs."""
+    __tablename__ = 'access_request'
+    id = Column(Integer, primary_key=True)
+
+    table_id = Column(Integer, ForeignKey('tables.id'))
+    druid_datasource_id = Column(Integer, ForeignKey('datasources.id'))
```

Comment:
we probably want to follow the new pattern of `datasource_id` and `datasource_type` that was introduced in the Yahoo PR